### PR TITLE
Vb/add roles to users

### PR DESCRIPTION
**Pull request by Besermenji**
Please tell me if this is okay and if I need to change something.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -24,6 +31,52 @@ def admin?
...
+
+  def revoke_assembly_president
+    self.roles.delete(Role.assembly_president)
+  end
+
+  def assembly_president?
+    role?(:assembly_president)
+  end
+
+  # FACEBOOK TUFF
```
Comment (Besermenji) [inline app/models/user.rb]: A typo here, FACEBOOK STUFF. lol

File: app/models/user.rb [hunk 2]
```diff
@@ -1,4 +1,9 @@
 class User < ActiveRecord::Base
+  # will citizen role be a default one?
+  #   how will we add other roles to users?
+  #   Should addmin add those roles?
```
Comment (Besermenji) [inline app/models/user.rb]: I ment admin

File: app/models/role.rb [hunk 3]
```diff
@@ -6,4 +6,15 @@ def self.admin
     find_or_create_by(:name => "admin")
   end
 
+  def self.citizen
+  	find_or_create_by(:name => "citizen")
```
Comment (nikolalsvk) [inline app/models/role.rb]: Just fix indentation here.

**Comment by nikolalsvk** [comment]
Nice job :clap:

**Pull request merged**